src/llmcompressor/modifiers/quantization/autoround/__init__.py

```diff
@@ -0,0 +1,3 @@
+# ruff: noqa
+
```

Comment:
Can we move the autoround folder to be under modifiers? We will be removing the quantization folder soon